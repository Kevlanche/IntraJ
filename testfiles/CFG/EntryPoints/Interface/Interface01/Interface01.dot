digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[4]"}
{ rank = same; "InterfaceDecl[5]"; "ClassDecl[27]"}
{ rank = same; "Modifiers[6]"; "List[8]"; "List[9]"; "Modifiers[28]"; "Opt[31]"; "List[32]"; "List[33]"}
{ rank = same; "List[7]"; "FieldDecl[10]"; "MethodDecl[19]"; "List[29]"; "MethodDecl[34]"}
{ rank = same; "Modifiers[11]"; "PrimitiveTypeAccess[13]"; "List[14]"; "Entry[20]"; "Modifiers[21]"; "PrimitiveTypeAccess[23]"; "List[24]"; "List[25]"; "Opt[26]"; "Exit[27]"; "Modifier[30]"; "Entry[35]"; "Modifiers[36]"; "PrimitiveTypeAccess[40]"; "List[41]"; "List[47]"; "Opt[48]"; "Exit[49]"}
{ rank = same; "List[12]"; "FieldDeclarator[15]"; "List[22]"; "List[37]"; "ParameterDeclaration[42]"; "Block[49]"}
{ rank = same; "List[16]"; "Opt[17]"; "Modifier[38]"; "Modifier[39]"; "Modifiers[43]"; "ArrayTypeAccess[45]"; "List[50]"}
{ rank = same; "IntegerLiteral[18]"; "List[44]"; "TypeAccess[46]"; "VarDeclStmt[51]"; "ExprStmt[96]"}
{ rank = same; "Modifiers[52]"; "TypeAccess[54]"; "List[55]"; "Dot[97]"}
{ rank = same; "List[53]"; "VariableDeclarator[56]"; "VarAccess[98]"; "MethodAccess[99]"}
{ rank = same; "List[57]"; "Opt[58]"; "List[101]"; "List[102]"; "List[103]"; "UncheckedExceptions[100]"}
{ rank = same; "ClassInstanceExpr[59]"}
{ rank = same; "TypeAccess[60]"; "List[61]"; "Opt[62]"; "List[94]"; "List[95]"}
{ rank = same; "AnonymousDecl[63]"}
{ rank = same; "Modifiers[64]"; "List[66]"}
{ rank = same; "List[65]"; "MethodDecl[67]"}
{ rank = same; "Entry[68]"; "Modifiers[69]"; "PrimitiveTypeAccess[75]"; "List[76]"; "List[77]"; "Opt[78]"; "Exit[79]"}
{ rank = same; "List[70]"; "Block[79]"}
{ rank = same; "Annotation[71]"; "Modifier[74]"; "List[80]"}
{ rank = same; "TypeAccess[72]"; "List[73]"; "ExprStmt[81]"}
{ rank = same; "Dot[82]"}
{ rank = same; "Dot[83]"; "MethodAccess[86]"}
{ rank = same; "TypeAccess[84]"; "VarAccess[85]"; "List[88]"; "List[92]"; "List[93]"; "UncheckedExceptions[87]"}
{ rank = same; "AddExpr[89]"}
{ rank = same; "StringLiteral[90]"; "VarAccess[91]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[4]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"InterfaceDecl[5]" [label="InterfaceDecl\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[6]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[7]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[8]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[9]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"FieldDecl[10]" [label="FieldDecl\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[11]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[12]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[13]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[14]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"FieldDeclarator[15]" [label="FieldDeclarator\n < x >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[16]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[17]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"IntegerLiteral[18]" [label="IntegerLiteral\n < 21 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[19]" [label="MethodDecl\n < getAge >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[20]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[21]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[22]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[23]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[24]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[25]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[26]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Exit[27]" [label="Exit\n ", fillcolor=white   style=filled  ]
"ClassDecl[27]" [label="ClassDecl\n < Interface01 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[28]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[29]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[30]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[31]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[32]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[33]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[34]" [label="MethodDecl\n < main >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[35]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[36]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[37]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[38]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[39]" [label="Modifier\n < static >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[40]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[41]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ParameterDeclaration[42]" [label="ParameterDeclaration\n < args >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[43]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[44]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ArrayTypeAccess[45]" [label="ArrayTypeAccess\n < String >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[46]" [label="TypeAccess\n < String >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[47]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[48]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[49]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[50]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[51]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[52]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[53]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[54]" [label="TypeAccess\n < Age >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[55]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[56]" [label="VariableDeclarator\n < Age oj1 >", fillcolor=white   style=filled  ]
"List[57]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[58]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassInstanceExpr[59]" [label="ClassInstanceExpr\n <Age>", fillcolor=white   style=filled  ]
"TypeAccess[60]" [label="TypeAccess\n < Age >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[61]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[62]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AnonymousDecl[63]" [label="AnonymousDecl\n < Anonymous0 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[64]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[65]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[66]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[67]" [label="MethodDecl\n < getAge >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[68]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[69]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[70]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Annotation[71]" [label="Annotation\n < annotation >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[72]" [label="TypeAccess\n < Override >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[73]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[74]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[75]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[76]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[77]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[78]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[79]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[80]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[81]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[82]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[83]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[84]" [label="TypeAccess\n < System >", fillcolor=white   style=filled  ]
"VarAccess[85]" [label="VarAccess\n < out >", fillcolor=white   style=filled  ]
"MethodAccess[86]" [label="MethodAccess\n < print >", fillcolor=white   style=filled  ]
"List[88]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AddExpr[89]" [label="AddExpr\n < + >", fillcolor=white   style=filled  ]
"StringLiteral[90]" [label="StringLiteral\n < Age is  >", fillcolor=white   style=filled  ]
"VarAccess[91]" [label="VarAccess\n < x >", fillcolor=white   style=filled  ]
"List[92]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[93]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[87]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Exit[79]" [label="Exit\n ", fillcolor=white   style=filled  ]
"List[94]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[95]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[96]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[97]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarAccess[98]" [label="VarAccess\n < oj1 >", fillcolor=white   style=filled  ]
"MethodAccess[99]" [label="MethodAccess\n < getAge >", fillcolor=white   style=filled  ]
"List[101]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[102]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[103]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[100]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Exit[49]" [label="Exit\n ", fillcolor=white   style=filled  ]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"Modifiers[6]" -> "List[7]" [style=dashed, color=gray]
"InterfaceDecl[5]" -> "Modifiers[6]" [style=dashed, color=gray]
"InterfaceDecl[5]" -> "List[8]" [style=dashed, color=gray]
"Modifiers[11]" -> "List[12]" [style=dashed, color=gray]
"FieldDecl[10]" -> "Modifiers[11]" [style=dashed, color=gray]
"FieldDecl[10]" -> "PrimitiveTypeAccess[13]" [style=dashed, color=gray]
"FieldDeclarator[15]" -> "List[16]" [style=dashed, color=gray]
"Opt[17]" -> "IntegerLiteral[18]" [style=dashed, color=gray]
"FieldDeclarator[15]" -> "Opt[17]" [style=dashed, color=gray]
"List[14]" -> "FieldDeclarator[15]" [style=dashed, color=gray]
"FieldDecl[10]" -> "List[14]" [style=dashed, color=gray]
"List[9]" -> "FieldDecl[10]" [style=dashed, color=gray]
"MethodDecl[19]" -> "Entry[20]" [style=dashed, color=gray]
"Modifiers[21]" -> "List[22]" [style=dashed, color=gray]
"MethodDecl[19]" -> "Modifiers[21]" [style=dashed, color=gray]
"MethodDecl[19]" -> "PrimitiveTypeAccess[23]" [style=dashed, color=gray]
"MethodDecl[19]" -> "List[24]" [style=dashed, color=gray]
"MethodDecl[19]" -> "List[25]" [style=dashed, color=gray]
"MethodDecl[19]" -> "Opt[26]" [style=dashed, color=gray]
"MethodDecl[19]" -> "Exit[27]" [style=dashed, color=gray]
"List[9]" -> "MethodDecl[19]" [style=dashed, color=gray]
"InterfaceDecl[5]" -> "List[9]" [style=dashed, color=gray]
"List[4]" -> "InterfaceDecl[5]" [style=dashed, color=gray]
"List[29]" -> "Modifier[30]" [style=dashed, color=gray]
"Modifiers[28]" -> "List[29]" [style=dashed, color=gray]
"ClassDecl[27]" -> "Modifiers[28]" [style=dashed, color=gray]
"ClassDecl[27]" -> "Opt[31]" [style=dashed, color=gray]
"ClassDecl[27]" -> "List[32]" [style=dashed, color=gray]
"MethodDecl[34]" -> "Entry[35]" [style=dashed, color=gray]
"List[37]" -> "Modifier[38]" [style=dashed, color=gray]
"List[37]" -> "Modifier[39]" [style=dashed, color=gray]
"Modifiers[36]" -> "List[37]" [style=dashed, color=gray]
"MethodDecl[34]" -> "Modifiers[36]" [style=dashed, color=gray]
"MethodDecl[34]" -> "PrimitiveTypeAccess[40]" [style=dashed, color=gray]
"Modifiers[43]" -> "List[44]" [style=dashed, color=gray]
"ParameterDeclaration[42]" -> "Modifiers[43]" [style=dashed, color=gray]
"ArrayTypeAccess[45]" -> "TypeAccess[46]" [style=dashed, color=gray]
"ParameterDeclaration[42]" -> "ArrayTypeAccess[45]" [style=dashed, color=gray]
"List[41]" -> "ParameterDeclaration[42]" [style=dashed, color=gray]
"MethodDecl[34]" -> "List[41]" [style=dashed, color=gray]
"MethodDecl[34]" -> "List[47]" [style=dashed, color=gray]
"Modifiers[52]" -> "List[53]" [style=dashed, color=gray]
"VarDeclStmt[51]" -> "Modifiers[52]" [style=dashed, color=gray]
"VarDeclStmt[51]" -> "TypeAccess[54]" [style=dashed, color=gray]
"VariableDeclarator[56]" -> "List[57]" [style=dashed, color=gray]
"ClassInstanceExpr[59]" -> "TypeAccess[60]" [style=dashed, color=gray]
"ClassInstanceExpr[59]" -> "List[61]" [style=dashed, color=gray]
"Modifiers[64]" -> "List[65]" [style=dashed, color=gray]
"AnonymousDecl[63]" -> "Modifiers[64]" [style=dashed, color=gray]
"MethodDecl[67]" -> "Entry[68]" [style=dashed, color=gray]
"Annotation[71]" -> "TypeAccess[72]" [style=dashed, color=gray]
"Annotation[71]" -> "List[73]" [style=dashed, color=gray]
"List[70]" -> "Annotation[71]" [style=dashed, color=gray]
"List[70]" -> "Modifier[74]" [style=dashed, color=gray]
"Modifiers[69]" -> "List[70]" [style=dashed, color=gray]
"MethodDecl[67]" -> "Modifiers[69]" [style=dashed, color=gray]
"MethodDecl[67]" -> "PrimitiveTypeAccess[75]" [style=dashed, color=gray]
"MethodDecl[67]" -> "List[76]" [style=dashed, color=gray]
"MethodDecl[67]" -> "List[77]" [style=dashed, color=gray]
"Dot[83]" -> "TypeAccess[84]" [style=dashed, color=gray]
"Dot[83]" -> "VarAccess[85]" [style=dashed, color=gray]
"Dot[82]" -> "Dot[83]" [style=dashed, color=gray]
"AddExpr[89]" -> "StringLiteral[90]" [style=dashed, color=gray]
"AddExpr[89]" -> "VarAccess[91]" [style=dashed, color=gray]
"List[88]" -> "AddExpr[89]" [style=dashed, color=gray]
"MethodAccess[86]" -> "List[88]" [style=dashed, color=gray]
"MethodAccess[86]" -> "List[92]" [style=dashed, color=gray]
"MethodAccess[86]" -> "List[93]" [style=dashed, color=gray]
"MethodAccess[86]" -> "UncheckedExceptions[87]" [style=dashed, color=gray]
"Dot[82]" -> "MethodAccess[86]" [style=dashed, color=gray]
"ExprStmt[81]" -> "Dot[82]" [style=dashed, color=gray]
"List[80]" -> "ExprStmt[81]" [style=dashed, color=gray]
"Block[79]" -> "List[80]" [style=dashed, color=gray]
"Opt[78]" -> "Block[79]" [style=dashed, color=gray]
"MethodDecl[67]" -> "Opt[78]" [style=dashed, color=gray]
"MethodDecl[67]" -> "Exit[79]" [style=dashed, color=gray]
"List[66]" -> "MethodDecl[67]" [style=dashed, color=gray]
"AnonymousDecl[63]" -> "List[66]" [style=dashed, color=gray]
"Opt[62]" -> "AnonymousDecl[63]" [style=dashed, color=gray]
"ClassInstanceExpr[59]" -> "Opt[62]" [style=dashed, color=gray]
"ClassInstanceExpr[59]" -> "List[94]" [style=dashed, color=gray]
"ClassInstanceExpr[59]" -> "List[95]" [style=dashed, color=gray]
"Opt[58]" -> "ClassInstanceExpr[59]" [style=dashed, color=gray]
"VariableDeclarator[56]" -> "Opt[58]" [style=dashed, color=gray]
"List[55]" -> "VariableDeclarator[56]" [style=dashed, color=gray]
"VarDeclStmt[51]" -> "List[55]" [style=dashed, color=gray]
"List[50]" -> "VarDeclStmt[51]" [style=dashed, color=gray]
"Dot[97]" -> "VarAccess[98]" [style=dashed, color=gray]
"MethodAccess[99]" -> "List[101]" [style=dashed, color=gray]
"MethodAccess[99]" -> "List[102]" [style=dashed, color=gray]
"MethodAccess[99]" -> "List[103]" [style=dashed, color=gray]
"MethodAccess[99]" -> "UncheckedExceptions[100]" [style=dashed, color=gray]
"Dot[97]" -> "MethodAccess[99]" [style=dashed, color=gray]
"ExprStmt[96]" -> "Dot[97]" [style=dashed, color=gray]
"List[50]" -> "ExprStmt[96]" [style=dashed, color=gray]
"Block[49]" -> "List[50]" [style=dashed, color=gray]
"Opt[48]" -> "Block[49]" [style=dashed, color=gray]
"MethodDecl[34]" -> "Opt[48]" [style=dashed, color=gray]
"MethodDecl[34]" -> "Exit[49]" [style=dashed, color=gray]
"List[33]" -> "MethodDecl[34]" [style=dashed, color=gray]
"ClassDecl[27]" -> "List[33]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[27]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[4]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"Exit[27]" -> "Entry[20]" [color=red, constraint=false]
"Entry[20]" -> "Exit[27]" [color=blue, constraint=false, xlabel=" "] 
"Exit[49]" -> "MethodAccess[99]" [color=red, constraint=false]
"MethodAccess[99]" -> "Exit[49]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[99]" -> "VarAccess[98]" [color=red, constraint=false]
"VarAccess[98]" -> "MethodAccess[99]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[98]" -> "VariableDeclarator[56]" [color=red, constraint=false]
"VariableDeclarator[56]" -> "VarAccess[98]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[56]" -> "ClassInstanceExpr[59]" [color=red, constraint=false]
"ClassInstanceExpr[59]" -> "VariableDeclarator[56]" [color=blue, constraint=false, xlabel=" "] 
"ClassInstanceExpr[59]" -> "Entry[35]" [color=red, constraint=false]
"Entry[35]" -> "ClassInstanceExpr[59]" [color=blue, constraint=false, xlabel=" "] 
"Exit[79]" -> "MethodAccess[86]" [color=red, constraint=false]
"MethodAccess[86]" -> "Exit[79]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[86]" -> "AddExpr[89]" [color=red, constraint=false]
"AddExpr[89]" -> "MethodAccess[86]" [color=blue, constraint=false, xlabel=" "] 
"AddExpr[89]" -> "VarAccess[91]" [color=red, constraint=false]
"VarAccess[91]" -> "AddExpr[89]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[91]" -> "StringLiteral[90]" [color=red, constraint=false]
"StringLiteral[90]" -> "VarAccess[91]" [color=blue, constraint=false, xlabel=" "] 
"StringLiteral[90]" -> "VarAccess[85]" [color=red, constraint=false]
"VarAccess[85]" -> "StringLiteral[90]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[85]" -> "TypeAccess[84]" [color=red, constraint=false]
"TypeAccess[84]" -> "VarAccess[85]" [color=blue, constraint=false, xlabel=" "] 
"TypeAccess[84]" -> "Entry[68]" [color=red, constraint=false]
"Entry[68]" -> "TypeAccess[84]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[4]" [style=invis]
"InterfaceDecl[5]" -> "ClassDecl[27]" [style=invis]
"Modifiers[6]" -> "List[8]" -> "List[9]" [style=invis]
"FieldDecl[10]" -> "MethodDecl[19]" [style=invis]
"Modifiers[11]" -> "PrimitiveTypeAccess[13]" -> "List[14]" [style=invis]
"List[16]" -> "Opt[17]" [style=invis]
"Entry[20]" -> "Modifiers[21]" -> "PrimitiveTypeAccess[23]" -> "List[24]" -> "List[25]" -> "Opt[26]" -> "Exit[27]" [style=invis]
"Modifiers[28]" -> "Opt[31]" -> "List[32]" -> "List[33]" [style=invis]
"Entry[35]" -> "Modifiers[36]" -> "PrimitiveTypeAccess[40]" -> "List[41]" -> "List[47]" -> "Opt[48]" -> "Exit[49]" [style=invis]
"Modifier[38]" -> "Modifier[39]" [style=invis]
"Modifiers[43]" -> "ArrayTypeAccess[45]" [style=invis]
"VarDeclStmt[51]" -> "ExprStmt[96]" [style=invis]
"Modifiers[52]" -> "TypeAccess[54]" -> "List[55]" [style=invis]
"List[57]" -> "Opt[58]" [style=invis]
"TypeAccess[60]" -> "List[61]" -> "Opt[62]" [style=invis]
"Modifiers[64]" -> "List[66]" [style=invis]
"Entry[68]" -> "Modifiers[69]" -> "PrimitiveTypeAccess[75]" -> "List[76]" -> "List[77]" -> "Opt[78]" -> "Exit[79]" [style=invis]
"Annotation[71]" -> "Modifier[74]" [style=invis]
"TypeAccess[72]" -> "List[73]" [style=invis]
"Dot[83]" -> "MethodAccess[86]" [style=invis]
"TypeAccess[84]" -> "VarAccess[85]" [style=invis]
"StringLiteral[90]" -> "VarAccess[91]" [style=invis]
"VarAccess[98]" -> "MethodAccess[99]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
107", fillcolor=1]
Edges [label="Total Nr. Edges
132", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
14", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
13", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}