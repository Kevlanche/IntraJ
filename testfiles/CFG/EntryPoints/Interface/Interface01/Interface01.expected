NodeName: Entry[20] Successors: [Exit[27]]
NodeName: Exit[27] Successors: []
NodeName: Entry[35] Successors: [ClassInstanceExpr[59]]
NodeName: ClassInstanceExpr[59] Successors: [VariableDeclarator[56]]
NodeName: VariableDeclarator[56] Successors: [VarAccess[98]]
NodeName: VarAccess[98] Successors: [MethodAccess[99]]
NodeName: MethodAccess[99] Successors: [Exit[49]]
NodeName: Exit[49] Successors: []
NodeName: Entry[68] Successors: [TypeAccess[84]]
NodeName: TypeAccess[84] Successors: [VarAccess[85]]
NodeName: VarAccess[85] Successors: [StringLiteral[90]]
NodeName: StringLiteral[90] Successors: [VarAccess[91]]
NodeName: VarAccess[91] Successors: [AddExpr[89]]
NodeName: AddExpr[89] Successors: [MethodAccess[86]]
NodeName: MethodAccess[86] Successors: [Exit[79]]
NodeName: Exit[79] Successors: []

NodeName: Exit[27] Predecessor: [Entry[20]]
NodeName: Entry[20] Predecessor: []
NodeName: Exit[49] Predecessor: [MethodAccess[99]]
NodeName: MethodAccess[99] Predecessor: [VarAccess[98]]
NodeName: VarAccess[98] Predecessor: [VariableDeclarator[56]]
NodeName: VariableDeclarator[56] Predecessor: [ClassInstanceExpr[59]]
NodeName: ClassInstanceExpr[59] Predecessor: [Entry[35]]
NodeName: Entry[35] Predecessor: []
NodeName: Exit[79] Predecessor: [MethodAccess[86]]
NodeName: MethodAccess[86] Predecessor: [AddExpr[89]]
NodeName: AddExpr[89] Predecessor: [VarAccess[91]]
NodeName: VarAccess[91] Predecessor: [StringLiteral[90]]
NodeName: StringLiteral[90] Predecessor: [VarAccess[85]]
NodeName: VarAccess[85] Predecessor: [TypeAccess[84]]
NodeName: TypeAccess[84] Predecessor: [Entry[68]]
NodeName: Entry[68] Predecessor: []
